ebs_snapshots/snapshot_manager.py

```diff
@@ -67,10 +68,10 @@ def _ensure_snapshot(connection, volume, interval, name):
     :returns: None
     """
     if interval not in VALID_INTERVALS:
-        print kayvee.formatLog("ebs-snapshots", "warning", "invalid snapshotting interval", {
+        logging.info(kayvee.formatLog("ebs-snapshots", "warning", "invalid snapshotting interval", {
```

Comment:
Will make it match. Ideally we'll someday say `kayvee.<level>` and remove the formatLog too